app/lunatech/lunchplanner/controllers/ReportController.scala

```diff
@@ -37,14 +37,19 @@ class ReportController @Inject()(userService: UserService,
   val year = "year"
 
   def getReport: EssentialAction = userAction.async { implicit request =>
-    val reportMonth = request.session.get(month).map(_.toInt).getOrElse(getDefaultDate.month)
-    val reportYear = request.session.get(year).map(_.toInt).getOrElse(getDefaultDate.year)
+    val reportMonth =
+      request.session.get(month).map(_.toInt).getOrElse(getDefaultDate.month)
```

Comment:
I understand that this isn't part of your changes, but I noticed that we can make this simpler. We can create an issue regarding further cleaning. XD

I would suggest making a function of `String => Int` or something like that.

Reply:
I also see other code that really need improvement, there's a stupid boolean calculation in there. I would prefer to change it in another PR, it don't want to add unnecessary changes to this one.